### PR TITLE
Actually do what the comment says

### PR DESCRIPTION
**Pull request by tcharding**
During rebasing and merging 3 different PRs I managed to leave a line
of code in that was not intended, this is both wrong and is contrary
to what the code comment says.

This is a totally trivial oneline fix, first reviewer feel free to kick bors to save the next reviewer the time.

### CODE REVIEW

**Comment by luckysori** [comment]
@tcharding:

Not sure if it's related, but you have a build error:
```
error[E0432]: unresolved import `crate::network::SwapExists`
 --> cnd/src/http_api/problem.rs:9:5
  |
9 |     network::SwapExists,
  |     ^^^^^^^^^^^^^^^^^^^ no `SwapExists` in `network`
```

**Comment by tcharding** [comment]
> @tcharding:
> 
> Not sure if it's related, but you have a build error:
> 
> ```
> error[E0432]: unresolved import `crate::network::SwapExists`
>  --> cnd/src/http_api/problem.rs:9:5
>   |
> 9 |     network::SwapExists,
>   |     ^^^^^^^^^^^^^^^^^^^ no `SwapExists` in `network`
> ```

Oh, I should be more careful.  Thanks.  This PR is totally wrong, I mis-read the network `comit` module for the comit crate. Its just the comment thats in the wrong place, hello whitespace patch.

File: cnd/src/network.rs [hunk 1]
```diff
@@ -4,8 +4,8 @@ pub mod start_swap;
 pub mod test_swarm;
 pub mod transport;
 
-// Export comit network types while maintaining the module abstraction.
 pub use self::comit::*;
+// Export comit network types while maintaining the module abstraction.
```
Comment (D4nte) [inline cnd/src/network.rs:8]: Maybe just remove the comment?

**Pull request merged**